### PR TITLE
[WEEX-581][android] fix 'onFocus invoked twice

### PR DESCRIPTION
**Pull request by chenzefeng09**
when <input> label has both '@focus' & '@blur' method, both methods will be invoked twice when trigger focus/blur event

### CODE REVIEW

**Comment by YorkShen** [comment]
Please rebase your commit, there is a conflict. @chenzefeng09

**Comment by YorkShen** [comment]
@gubaojian Need your approval, thanks

File: android/sdk/src/main/java/com/taobao/weex/ui/component/WXComponent.java [hunk 1]
```diff
@@ -175,6 +175,7 @@
   public boolean mIsAddElementToTree = false;
   //for fix element case
   public int interactionAbsoluteX=0,interactionAbsoluteY=0;
+  public boolean mHasAddFocusListener = false;
```
Comment (YorkShen) [inline android/sdk/src/main/java/com/taobao/weex/ui/component/WXComponent.java]: Sorry for the delay.

Make `mHasAddFocusListener` to private, thanks.

Comment (chenzefeng09) [reply android/sdk/src/main/java/com/taobao/weex/ui/component/WXComponent.java]: done

**Pull request merged**